snapcraft.yaml

```diff
@@ -42,9 +42,33 @@ parts:
     plugin: nil
     build-packages:
       - sbsigntool
+      - wget
+      - attr
     stage-packages:
       - grub-efi-$CRAFT_ARCH_BUILD_FOR-signed
-      - shim-signed
```

Comment:
Maybe comment instead of removing and add a TODO to revert things when 2.63 is out

Reply:
I have added a TODO in the override-pull